go/cmd/dolt/commands/diff.go

```diff
@@ -956,8 +956,7 @@ func diffSummary(ctx context.Context, td diff.TableDelta, colLen int) errhand.Ve
 	acc := diff.DiffSummaryProgress{}
 	var count int64
 	var pos int
-	eP := cli.StartEphemeralPrinter()
-	defer eP.Stop()
+	eP := cli.NewEphemeralPrinter()
```

Comment:
Nice job removing the need to call `Start` and `Stop`. The EP interface looks cleaner now from that. Also looks nice to have the more standard "New<object>" pattern instead of "Start<object>".